src/types/mod.rs

```diff
@@ -78,9 +79,8 @@ pub fn get_body_from_pyobject(body: &PyAny) -> PyResult<Vec<u8>> {
     } else if let Ok(b) = body.downcast::<PyBytes>() {
         Ok(b.as_bytes().to_vec())
     } else {
-        Err(PyValueError::new_err(
```

Comment:
Why did we remove this Error handling?

Reply:
Hey @StandinKP 👋 

This needs to be handled up the chain